pkg/util/util_supported.go

```diff
@@ -4,88 +4,10 @@
 package util
 
 import (
-	"errors"
-	"fmt"
-	"os"
-	"path/filepath"
-	"sync"
-	"syscall"
-
 	"github.com/containers/storage/pkg/homedir"
-	"github.com/containers/storage/pkg/unshare"
-	"github.com/sirupsen/logrus"
-)
-
-var (
-	rootlessRuntimeDirOnce sync.Once
-	rootlessRuntimeDir     string
 )
 
-// isWriteableOnlyByOwner checks that the specified permission mask allows write
-// access only to the owner.
-func isWriteableOnlyByOwner(perm os.FileMode) bool {
-	return (perm & 0o722) == 0o700
-}
-
 // GetRuntimeDir returns the runtime directory
 func GetRuntimeDir() (string, error) {
```

Comment:
This is fine for a test PR; when the current project is to consolidate, I think it would be better to migrate all callers to call the c/storage version directly, so that future maintainers are less tempted to introduce divergence again.